pkg/resource/logging-config/alloy/test/logging-config.alloy.170_WC_custom_tenants.yaml

```diff
@@ -0,0 +1,353 @@
+        	}
+        }
+        
+        loki.source.journal "systemd_journal_var" {
+        	format_as_json = true
+        	max_age        = "12h0m0s"
+        	path           = "/var/log/journal"
+        	relabel_rules  = discovery.relabel.systemd_journal_var.rules
+        	forward_to     = [loki.process.systemd_journal_var.receiver]
+        	labels         = {
+        		scrape_job = "system-logs",
+        	}
+        }
+        
+        // Kubernetes API server audit logs
+        local.file_match "kubernetes_audit" {
+        	path_targets = [{
+        		__address__ = "localhost",
+        		__path__    = "/var/log/apiserver/audit.log",
+        		node   = coalesce(env("NODENAME"), "unknown"),
+        		scrape_job  = "audit-logs",
+        	}]
+        }
+        
+        loki.process "kubernetes_audit" {
+        	forward_to = [loki.write.default.receiver]
+        
+        	stage.json {
+        		expressions = {
+        			objectRef = "objectRef",
+        		}
+        	}
+        
+        	stage.json {
+        		expressions = {
+        			namespace = "namespace",
+        			resource  = "resource",
+        		}
+        		source = "objectRef"
+        	}
+        
+        	stage.structured_metadata {
+        		values = {
+        			"resource" = "",
+        			"filename" = "",
+        		}
+        	}
+        
+        	stage.label_drop {
+        		values = [
+        			"filename",
+        		]
+        	}
+        
+        	stage.labels {
+        		values = {
+        			namespace = "",
+        		}
+        	}
+        }
+        
+        loki.source.file "kubernetes_audit" {
+        	targets               = local.file_match.kubernetes_audit.targets
+        	forward_to            = [loki.process.kubernetes_audit.receiver]
+        	legacy_positions_file = "/run/promtail/positions.yaml"
+        }
+        
+        // Loki target configuration
+        loki.write "default" {
+        	endpoint {
+        		url                = nonsensitive(remote.kubernetes.secret.credentials.data["logging-url"])
+        		max_backoff_period = "10m"
+        		tenant_id          = nonsensitive(remote.kubernetes.secret.credentials.data["logging-tenant-id"])
+        
+        		basic_auth {
+        			username = nonsensitive(remote.kubernetes.secret.credentials.data["logging-username"])
+        			password = remote.kubernetes.secret.credentials.data["logging-password"]
+        		}
+        
+        		tls_config {
+        			insecure_skip_verify = false
+        		}
+        	}
+        	external_labels = {
+        		cluster_id   = "test-cluster",
+        		installation = "test-installation",
+        	}
+        }
+        
+        logging {
+        	level  = "warn"
+        	format = "logfmt"
+        }
+        
+    clustering:
+      enabled: true
+      name: alloy-logs
+    extraEnv:
+    - name: NODENAME
+      valueFrom:
+        fieldRef:
+          fieldPath: spec.nodeName
+    mounts:
+      varlog: true
```

Comment:
Why did this file change fully?

Reply:
That's a new file for a new test case with multiple tenants